src/card.css

```diff
@@ -0,0 +1,139 @@
+/* stylelint-disable string-quotes */
+/* stylelint-disable color-hex-length */
+/* stylelint-disable declaration-block-trailing-semicolon */
+/* stylelint-disable shorthand-property-no-redundant-values */
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap');
+
+.product-card {
+  display: block;
+  background-position: center;
+  height: 406px;
+  width: 198px;
+  margin: 0 auto;
+  border: 1px solid #f3f3f3;
+  justify-content: center;
+  align-items: center;
+
+}
+
+.product-image {
+  position: flex;
+  background-image: url("images/imac.jpeg");
+  background-size: cover;
+  background-position: center;
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px 19px;
+}
+
+.product-image img {
+  max-width: 100%;
+  max-height: 100%;
+  display: block;
+}
+
+.product-name {
+  width: 166px;
+  height: 36px;
```

Comment:
![image](https://user-images.githubusercontent.com/80700247/229275215-2665f496-d158-414e-b9c6-3accba2c98b5.png)

Also in this case you dont need to hardcode with, just center whole your card content and add necessary paddings

Fix everywhere